packages/utils/src/did-utils.ts

```diff
@@ -138,7 +138,7 @@ function extractPublicKeyBytes(pk: VerificationMethod): Uint8Array {
   if (pk.publicKeyBase58) {
     return base58ToBytes(pk.publicKeyBase58)
   } else if (pk.publicKeyMultibase) {
-    return bases['base58btc'].decode(pk.publicKeyMultibase)
```

Comment:
this line should be refactored out to a utility function, similar to `hexToMultibase`. Should probably live in the same file as that function, `packages/utils/src/encodings.ts`